llvm/include/llvm/CodeGen/AsmPrinter.h

```diff
@@ -207,9 +208,9 @@ class LLVM_ABI AsmPrinter : public MachineFunctionPass {
     using CGTypeId = uint64_t;
 
     /// Unique target type IDs.
-    SmallSet<CGTypeId, 4> IndirectCalleeTypeIDs;
+    SetVector<CGTypeId> IndirectCalleeTypeIDs;
     /// Unique direct callees.
-    SmallSet<MCSymbol *, 4> DirectCallees;
+    SetVector<MCSymbol *> DirectCallees;
```

Comment:
```suggestion
    SmallSetVector<MCSymbol *, 4> DirectCallees;
```

Started Small so keep as Small?

Reply:
oops. good catch. thanks.